rsts/getting_started.rst

```diff
@@ -1,109 +1,345 @@
+
+           .. prompt::
+
+               flytectl sandbox exec -- docker build . --tag "myapp:v1"
+
+           .. tip::
+            #. Why are we not pushing the docker image? Want to understand the details —— refer to guide `here <>`_
+            #. *Recommended:* Use the bundled `./docker_build_and_tag.sh`. It will automatically build the local Dockerfile, name it and tag it with the current git-SHA. This helps in achieving GitOps style workflows.
+
+       .. tab:: Remote Flyte Cluster
+
+           If you are using a remote Flyte cluster, then you need to build your container and push it to a registry that is accessible by the Flyte Kubernetes cluster.
+
+           .. prompt::
+
+               docker build . --tag registry/repo:version
+               docker push registry/repo:version
+
+#. Now that the container is built let's provide this information to the Flyte backend, for which you have to package the workflow using the ``pyflyte`` cli bundled with Flytekit. Also, note that the image is the same as the one built in the previous step. ::
+
+    pyflyte --pkgs myapp.workflows package --image myapp:v1
+
+#. Let's now upload this package to the Flyte backend. We refer to this as ``registration``. ::
+
+    flytectl register files -p flytesnacks -d development -a flyte-package.tgz  -v v1
+
+#. Visualize the registered workflow. ::
+
+    flytectl get workflows -p flytesnacks -d development myapp.workflows.example.my_wf -o doturl
+
+
+.. _getting-started-execute:
+
+Execute on Flyte
+^^^^^^^^^^^^^^^^
+
+Finally, use FlyteConsole to launch an execution and keep tabs on the window! 
 
 .. image:: https://raw.githubusercontent.com/flyteorg/flyte/static-resources/img/flytesnacks/tutorial/exercise.gif
     :alt: A quick visual tour for launching a workflow and checking the outputs when they're done.
 
-.. admonition:: Recap
+**Alternatively,** 
+
+Launch and monitor from CLI using flytectl. This is how you will have to proceed.
+More details can be found `here <https://docs.flyte.org/projects/flytectl/en/stable/gen/flytectl_create_execution.html>`__
+
+#. Generate execution spec file. ::
+
+    flytectl get launchplan -p flytesnacks -d development myapp.workflows.example.my_wf  --execFile exec_spec.yaml
+
+#. Update the input spec file for arguments to the workflow. ::
+
+            ....
+            inputs:
+              name: "adam"
+            ....
+
+#. Create execution using the exec spec file. ::
+
+    flytectl create execution -p flytesnacks -d development --execFile exec_spec.yaml
+
+
+#. Monitor the execution by providing the execution id from create command. ::
+
+    flytectl get execution -p flytesnacks -d development <execid>
+
+
+
+For the Explorers
+*****************
+
+If you're interested in poking around the various ways to make running code on Flyte comfortable, look further.
```

Comment:
What does comfortable even mean here? It's not immediately obvious what the purpose of the subsequent section is from this introduction & header